agent-control/src/agent_type/runtime_config/on_host/package/rendered.rs

```diff
@@ -0,0 +1,23 @@
+use crate::agent_type::runtime_config::on_host::package::PackageType;
+use oci_spec::distribution::Reference;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Package {
+    /// Package type (tar.gz, zip).
+    pub(super) package_type: PackageType,
+
+    /// Download defines the supported repository sources for the packages.
+    pub(super) download: Download,
```

Comment:
If we ever want to support anything else, we will only use one repository at once, right?
I'm asking because if that's the case, we should use an enum.

Or the idea is that we will have multiple sources and we will download from the first one?
Or is it still open?